CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## unreleased / ????-??-??
 
+* [ENHANCEMENT] Updated readme docs and added a test for invalid yaml ([#274](https://github.com/prometheus/statsd_exporter/issues/274))
```

Comment:
I usually don't include documentation and test changes in the changelog, because I want to focus on what is important for someone who is already using it in production. Could you remove this please?

Reply:
(the README is not released as part of the tarball, so it doesn't really have _versioning_ per se)

Reply:
Sure!